flink-connector-mongodb/src/main/java/org/apache/flink/connector/mongodb/source/reader/split/MongoScanSourceSplitReader.java

```diff
@@ -119,7 +119,6 @@ public RecordsWithSplitIds<BsonDocument> fetch() throws IOException {
             throw new IOException("Scan records form MongoDB failed", e);
         } finally {
             if (finished) {
-                currentSplit = null;
```

Comment:
What's the value that currentSplit should be assigned if a split has been finished ?

Reply:
Thanks @leonardBang for the review.
It should be null, but this line is redundant because it is also set to null in the `closeCursor` method.